storage/objects/get_metadata.go

```diff
@@ -64,6 +64,10 @@ func getMetadata(w io.Writer, bucket, object string) (*storage.ObjectAttrs, erro
 	fmt.Fprintf(w, "Temporary hold enabled? %t\n", attrs.TemporaryHold)
 	fmt.Fprintf(w, "Retention expiration time %v\n", attrs.RetentionExpirationTime)
 	fmt.Fprintf(w, "Custom time %v\n", attrs.CustomTime)
+	if attrs.Retention != nil {
```

Comment:
IMO just print this and remove the conditional.  Use `%+v` to include the field names in the struct.

Reply:
Eh, I don't think it matters that much. Honestly we should probably just remove all the specific fields from the sample and use `%+v` on the Attrs object (C++ does this...).